### PR TITLE
dig lookup-plugin: add support for specifying rdclass

### PR DESCRIPTION
**Pull request by jpmens**
##### SUMMARY
This enhances https://github.com/ansible/ansible/pull/10311 by allowing the class to be specified, e.g. for doing `version.bind` lookups.

##### ISSUE TYPE
Feature Pull Request

##### COMPONENT NAME
lib/ansible/plugins/lookup/dig.py

##### ANSIBLE VERSION
2.3

### CODE REVIEW

**Comment by amenonsen** [comment]
So it looks like the `dig` lookup plugin doesn't exist in v2/devel at all.

**Comment by bcoca** [comment]
not all lookup plugins have been ported yet

**Comment by sivel** [comment]
@jpmens,

Hi!

Thank you very much for your submission to Ansible.  It sincerely means a lot to us.

Unfortunately it looks like the code underneath has changed since this was submitted.  Specifically this change was written against a code path that no longer exists in Ansible.

We would like to ask if you could either update the code in this pull request to apply against the current state of Ansible or close this pull request. If we do not hear back from you, we will close this pull request out shortly.

Thank you once again for this and your interest in Ansible!

**Comment by jpmens** [comment]
Done

File: lib/ansible/plugins/lookup/dig.py [hunk 1]
```diff
@@ -156,6 +158,11 @@ def run(self, terms, variables=None, **kwargs):
                     qtype = arg.upper()
                 elif opt == 'flat':
                     flat = int(arg)
+                elif opt == 'class':
+                    try:
+                        rdclass = dns.rdataclass.from_text(arg)
+                    except Exception, e:
```
Comment (bcoca) [inline lib/ansible/plugins/lookup/dig.py]: ```Exception as e```

lookups need to be 2.6/3.5 compatible, not 2.4 (that is for modules)

Comment (jpmens) [reply lib/ansible/plugins/lookup/dig.py]: fixed

**Comment by jpmens** [comment]
I don't know what you want from me; this PR is dated TWO YEARS ago.

**Pull request merged**